app/src/main/java/kth/jjve/xfran/SaveResultsActivity.java

```diff
@@ -0,0 +1,157 @@
+package kth.jjve.xfran;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.google.android.material.navigation.NavigationView;
+
+import java.util.ArrayList;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.ActionBarDrawerToggle;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+import androidx.core.view.GravityCompat;
+import androidx.drawerlayout.widget.DrawerLayout;
+import kth.jjve.xfran.models.Workout;
+
+import static android.view.View.GONE;
+
+public class SaveResultsActivity extends AppCompatActivity implements NavigationView.OnNavigationItemSelectedListener {
+
+    private final String LOG_TAG = getClass().getSimpleName();
+
+    /*_________ INTENT _________*/
+    private int position;
```

Comment:
for next time I would user the wrapper class `Integer` instead of its primitive `int`. This gives you more functionabilities